src/storage/storage.cc

```diff
@@ -54,8 +54,18 @@ constexpr const char *kReplicationIdKey = "replication_id_";
 
 const int64_t kIORateLimitMaxMb = 1024000;
 
+const static std::vector<CompressionOption> compressionOptions = {
+    {rocksdb::kNoCompression, "no", "kNoCompression"},
+    {rocksdb::kSnappyCompression, "snappy", "kSnappyCompression"},
+    {rocksdb::kZlibCompression, "zlib", "kZlibCompression"},
+    {rocksdb::kLZ4Compression, "lz4", "kLZ4Compression"},
+    {rocksdb::kZSTD, "zstd", "kZSTD"},
+};
+
 using rocksdb::Slice;
 
+std::vector<CompressionOption> GetCompressionOptions() { return compressionOptions; }
```

Comment:
```suggestion
const std::vector<CompressionOption>& GetCompressionOptions() { return compressionOptions; }
```

Reply:
we do not need to construct a vector every time when we call the function, which is expensive. BTW I think it is no difference than just an `inline const` vector in header file.

Reply:
Yes, it's better to put this in the header file.